fake-koji/src/main/java/org/fakekoji/api/http/rest/GetterAPI.java

```diff
@@ -93,6 +93,7 @@
     private static final String UNSAFE = "unsafe";
     private static final String WEBAPP = "webapp";
     private static final String XML_RPC = "xmlRpc";
+    private static final String JENKINS = "jenkins";
```

Comment:
Cen you plese look into accessibleSettings.getPublicValues and comapre? Those two apis looks quite diverge now and should get more aligned.
Can you also add getter for whole jenkins url?
I can see in get/help there is /url/jenkins but afaik it is nto implemented. Tahx!

Reply:
getPublicValues hasn't been used since we added Javalin. I wanted to remove it along with ProjectMappings but that's still used in: https://github.com/judovana/jenkins-scm-koji-plugin/blob/dc0bf41c0f7fd115424520e3c40f0fbf577953c0/fake-koji/src/main/java/org/fakekoji/core/FakeBuild.java#L388 And I guess you don't want to get rid of supportedarches.

Actually, it is implemented :smile:: http://hydra.brq.redhat.com:8888/get/jenkinsUrl

Reply:
> getPublicValues hasn't been used since we added Javalin. I wanted to remove it along with ProjectMappings but that's still used in:
> 
> https://github.com/judovana/jenkins-scm-koji-plugin/blob/dc0bf41c0f7fd115424520e3c40f0fbf577953c0/fake-koji/src/main/java/org/fakekoji/core/FakeBuild.java#L388
> And I guess you don't want to get rid of supportedarches.

Then pelase rmeove everything what is not used.
> 
> Actually, it is implemented smile: http://hydra.brq.redhat.com:8888/get/jenkinsUrl

Then please fix the help message.

Reply:
ou...didn't notice the url is wrong...will fix it

I will leave ProjectMapping for another PR